File: Java/juegoTexto/Test_P2.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package juegoTexto;
import civitas.*;
import java.util.ArrayList;
/**
 *
 * @author ilzk
 */
public class Test_P2 {
    public static void main (String args[]) {
        VistaTextual vista = new VistaTextual();
        ArrayList<String> nombres = new ArrayList<>();
        nombres.add("Miguel");
        nombres.add("Roberro");
        nombres.add("Ca?as");
        nombres.add("Brian");
        CivitasJuego juego = new CivitasJuego(nombres);
        Dado.getInstance().setDebug(true);
        Controlador control = new Controlador(juego, vista);
        control.juega();
    }
}

File: Ruby/test_p2.rb
# To change this license header, choose License Headers in Project Properties.
# To change this template file, choose Tools | Templates
# and open the template in the editor.

require_relative 'civitas_juego'
require_relative 'controlador'
  
module JuegoTexto
  class Test_P2
    def main
      nombres = []
      for i in 1..4
        nombres[i] = Civitas::Jugador.new("j" + i.to_s)
      end
      juego = Civitas::Civitas_juego.new(1, nombres)
      Civitas::Dado.instance.set_debug(true)
      control = Controlador.new(juego, Vista_textual.new())
      control.juega()
    end
  end
  
  test = Test_P2.new
  test.main
end